src/product/product.controller.ts

```diff
@@ -0,0 +1,38 @@
+import { Response, Request } from "express-serve-static-core";
+import express from "express";
```

Comment:
I guess you could write it in that way:

```suggestion
import express, { Response, Request } from "express";
```

Reply:
fixed